### ISSUE TITLE
bug: youtube revanced splash screen is grey instead of black

### ISSUE THREAD

**Issue by ankitxcoding** [revanced/revanced-patches#2336]
### Type

Cosmetic

### Bug description

The splash screen of the youtube revanced app is grey instead of black. This is happening after the last update.

### Steps to reproduce

Step 1. Close & clear the youtube revanced from the ram.
Step 2. Open the app again
Step 3. Notice the color of the splash screen, i.e grey instead of black.

### Relevant log output

```shell
.
```

### Screenshots or videos

![Screenshot_2023-06-01-12-35-47-61_64ef5fc2000c1caa954c114bb372e1d5](https://github.com/revanced/revanced-patches/assets/103440538/73a1a8c0-790d-4005-a793-c890f78a8d7f)

### Additional context

Patch version v2.175.0
Revanced manager version 1.3.0

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Alternatively, the dynamic color currently used, can be changed to the background color specified in the patch options

**Comment by oSumAtrIX** [comment]
Yes. This is because the dynamic color is defaulting to the gray color set by YouTube. If the dynamic "dark" color is patched to the background color specified in the option, the dynamic color should be black after applying the theme patch when using dark mode

**Comment by oSumAtrIX** [comment]
I'd consider this a bug because the theme patch is supposed to color the background, which also includes the splash screen background. Is it possible to change the gray color which the dynamic color refers to when in dark mode?

**Comment by oSumAtrIX** [comment]
I believe for the time being, reverting the commit that introduced the dynamic color would be a good idea.

Currently, if you apply the theme patch, neither the default amoled black theme color is used, nor does the splash screen background color match the background of the app. Reverting it would allow dark theme users to use the correct background color by default, only light theme users would have an issue like originally
